fixCucumber.ps1

```diff
@@ -0,0 +1,9 @@
+If ($env:ruby_version -match "200") {
+    $path = "C:\Ruby200\lib\ruby\gems\2.0.0\gems\cucumber-2.0.0\lib\cucumber\formatter\ansicolor.rb"
+} Else {
+    $path = "C:\Ruby193\lib\ruby\gems\1.9.1\gems\cucumber-2.0.0\lib\cucumber\formatter\ansicolor.rb"
+}
+
+(Get-Content $path) -replace '\ST\DERR.puts %', '#nope' | Set-Content $path 
+(Get-Content $path) -replace '    Cucumber::', '#nope' | Set-Content $path
+(Get-Content $path) -replace '^Cucumber::', '#nope' | Set-Content $path
```

Comment:
It's more of a problem of how appveyor is implemented, i.e. it doesn't need any additional library to get colored output - just like on *nix - and cucumber thinks it is not possible on windows. See:
https://github.com/appveyor/ci/issues/373
http://help.appveyor.com/discussions/suggestions/197-support-ansi-color-codes
It's not dependent on ruby version.